enterprise/templates/enterprise/templatetags/course_modal.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n enterprise %}
 
-<div id="course-details-modal-{{ index }}" class="modal fade" role="dialog" aria-modal="true" aria-labelledby="modal-header-text-{{ index }}" aria-hidden="true" tabindex="-1">
+<div id="course-details-modal-{{ index }}" class="modal fade modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text-{{ index }}" aria-hidden="true" tabindex="-1">
```

Comment:
Should `.modal-content` class be on the top-level `.modal`? There is a nested `div.modal-content` on line 5 below, too.